stable/spring-cloud-data-flow/templates/server-deployment.yaml

```diff
@@ -77,8 +77,8 @@ spec:
           secret:
             secretName: {{ printf "%s-%s" .Release.Name "mysql" | trunc 63 | trimSuffix "-" }}
       initContainers:
-      - name: init-mysql-wait
-        image: {{ .Values.mysql.busybox.image }}:{{ .Values.mysql.busybox.tag }}
-        imagePullPolicy: {{ .Values.mysql.imagePullPolicy }}
-        command: ['sh', '-c', 'until nc -w3 -z {{ printf "%s-%s" .Release.Name "mysql" | trunc 63 | trimSuffix "-" }} 3306; do echo waiting for mysql; sleep 3; done;']
+      - name: init-db-wait
+        image: {{ .Values.initContainers.dbWait.image }}:{{ .Values.initContainers.dbWait.tag }}
+        imagePullPolicy: {{ .Values.initContainers.dbWait.imagePullPolicy }}
+        command: ['sh', '-c', 'until nc -w3 -z {{ template "scdf.database.host" . }} {{ template "scdf.database.port" . }}; do echo waiting for mysql; sleep 3; done;']
```

Comment:
maybe "waiting for database" rather than "waiting for mysql" as the external DB could be something else?